### PR TITLE
feat: add TS linting check in CI

### LINKED ISSUE #346: (feat): add ts linting check in ci

### PR DESCRIPTION
**Pull request by viraj124**
- fixes existing ts linting errors
- closes #346

### CODE REVIEW

**Comment by DefiCake** [comment]
CIs are failing

**Comment by viraj124** [comment]
> CIs are failing



> CIs are failing

Yeah still wip

File: packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.fuel_message_portal_v3.ts [hunk 1]
```diff
@@ -25,10 +25,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     response.hash
   );
 
+  const implementation = receipt?.contractAddress ?? '';
+
+  if (implementation === '')
+    throw new Error(
+      `Upgrade proposement failed for FuelMessagePortalV3 proxy (${contractDeployment.address})`
```
Comment (PraneshASP) [inline packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.fuel_message_portal_v3.ts:32]: proposal*

File: packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.erc20_gateway_v4.ts [hunk 2]
```diff
@@ -25,10 +25,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     response.hash
   );
 
+  const implementation = receipt?.contractAddress ?? '';
+
+  if (implementation === '')
+    throw new Error(
+      `Upgrade proposement failed for FuelERC20GatewayV4 proxy (${contractDeployment.address})`
```
Comment (PraneshASP) [inline packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.erc20_gateway_v4.ts:32]: proposal*

File: packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.chain_state.ts [hunk 3]
```diff
@@ -25,10 +25,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     response.hash
   );
 
+  const implementation = receipt?.contractAddress ?? '';
+
+  if (implementation === '')
+    throw new Error(
+      `Upgrade proposement failed for FuelChainState proxy (${contractDeployment.address})`
```
Comment (PraneshASP) [inline packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.chain_state.ts:32]: proposal*

**Review by PraneshASP — CHANGES_REQUESTED** [review]
few typos

File: packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.fuel_message_portal_v3.ts (hunk 1 above)
Comment (viraj124) [reply packages/solidity-contracts/deploy/upgradeTest/prepareUpgrade.fuel_message_portal_v3.ts:32]: google said it was a legit word lol nvm will update

**Review by PraneshASP — APPROVED** [review]

**Pull request merged**